src/frontend/components/UI/SearchBar/index.tsx

```diff
@@ -24,15 +24,15 @@ export default React.memo(function SearchBar() {
   const input = useRef<HTMLInputElement>(null)
 
   const list = useMemo(() => {
-    const library = new Set(
-      [...epic.library, ...gog.library, ...sideloadedLibrary]
-        .filter(Boolean)
-        .map((g) => g.title)
-        .sort()
-    )
-    return [...library].filter((i) =>
-      new RegExp(fixFilter(filterText), 'i').test(i)
-    )
+    const library = [
```

Comment:
We are using Set right now to avoid duplicating items in case the person has the same game on Epic and Gog since we are using the title.

I think you can solve this issue by just doing the `epic?.library`, etc. Because we are already removing the invalid items when doing the `filter(Boolean)` later. I think that should do the trick.

Reply:
I added back the Set.